### ISSUE TITLE
Accuracy problem

### ISSUE THREAD

**Issue by xilaili** [D-X-Y/ResNeXt-DenseNet#7]
Hi,

I cannot reproduce the accuracy using your code. Which version of pytorch did you use to train the models?

Thanks.

**Comment by D-X-Y** [comment]
I test the code on both v0.1.12 and v0.2.0.
Can you provide more details? What scripts are you using to reproduce which model?

**Comment by xilaili** [comment]
python main.py ./data/cifar.python --dataset cifar100 --arch resnext29_8_64 --save_path ./snapshots/cifar10_resnext29_8_64_300 --epochs 300 0--schedule 150 225 --gammas 0.1 0.1 --batch_size 128 --workers 4

This is the command I used to test the model. I'm using pytorch v0.3.0. I got best test accuracy 81.46%.

**Comment by D-X-Y** [comment]
Please see https://github.com/D-X-Y/ResNeXt-DenseNet/blob/master/shells/cifar10_resnext29_8_64.sh for details.
To reproduce CIFAR results, you need to use 8 GPUs. To get comparable results using 2/4 GPUs, you should change the batch size and the corresponding learning rate. (more details at https://github.com/facebookresearch/ResNeXt)
